apps/dashboard/app/views/batch_connect/session_contexts/_path_selector.html.erb

```diff
@@ -39,30 +40,45 @@
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
-      <div class="modal-body">       
-        <div>
-          <ol id="<%= breadcrumb_id %>" class="breadcrumb breadcrumb-no-delimiter">
-          </ol>
+      <div class="modal-body">
+      
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-sm-5">
+              <div class="panel panel-default">
+                <div class="panel-heading">Favorites</div>
+                <div id="favorites" class="list-group text-nowrap">
+                  <%= render partial: 'favorites', collection: parse_favorites(file_picker_favorites), as: :path %>
+                </div>
```

Comment:
This should likely needs to be an `ol` and the partial `favorites` be `li` tags for accessibility.